### ISSUE TITLE
[Bug/Compatibility] Joining server message incorrect

### ISSUE THREAD

**Issue by voruti** [fooooooooooooooo/VelocityDiscord#11]
With the config line
```toml
[discord.chat]
join_message = "*{username} joined {server}*"
```
the wrong server is output in Discord. It shows the default server (lobby) instead of the server I'm actually joining.
I'm using this plugin to rejoin the last server I was connected to: https://github.com/Matt-MX/ReconnectVelocity

**Comment by klutzies** [comment]
That may have been my fault... (#3)
Could you try [this commit](https://github.com/klutzies/VelocityDiscord/commit/ff0e387572a97bc080e8f21fc7cb2016bad2b71f)? I switched the server connect methods from `PlayerChooseInitialServerEvent` and `ServerPostConnectEvent` to only `ServerConnectedEvent`, which may have something to do with your issue. Also, I haven't tested that change at all yet, and don't have time to do so right now, so if you could do that for me... ;)

**Comment by voruti** [comment]
Almost:
![screen](https://user-images.githubusercontent.com/45130096/200135084-b2345fa1-e91b-4f59-95b7-738764f566d6.png)

**Comment by unilock** [comment]
Yeah, I think I used `var server = event.getServer().getServerInfo();` instead of `var server = event.getServer().getServerInfo().getName();`. My bad. I've since rewritten the history of that branch, so let's pretend that never happened ;)

The fixed commit is here: https://github.com/unilock/VelocityDiscord/commit/c0115e6e81d9d0614738a215f5029065a37a60a1

Though I've continued working on that branch to implement a customizable webhook username, so I'll be opening a pull request regardless.
